app/controllers/v0/terms_of_use_agreements_controller.rb

```diff
@@ -25,7 +25,8 @@ def accept
 
     def accept_and_provision
```

Comment:
It won't let me add a comment to the line but we also need to rescue `TermsOfUse::Errors::ProvisionerError` here.


<img width="836" alt="Screenshot 2024-05-14 at 9 32 09 AM" src="https://github.com/department-of-veterans-affairs/vets-api/assets/10853209/42f863f3-37c4-44da-843b-716b7ce0eeeb">

Reply:
this should be fixed now